docs/UserGuide.md

```diff
@@ -97,14 +97,14 @@ Format: `adddebt <person index> d/<description> m/<money>`
 Example:
 * `adddebt 3 d/McDonalds m/8.9`
 
-### Deleting a debt: `deletedebt`
+### Clearing debts: `cleardebts`
 
 Deletes a debtor's debts from PayMeLah according to his index number when listed.
```

Comment:
Should this be updated to be consistent with the command's name? Maybe we should be specific that _all_ debts will be cleared.